#!/usr/bin/env python
PACKAGE = "auv_localization"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("yaw_offset", double_t, 0, "Yaw offset in radians", 2.34, -3.14, 3.14)

exit(gen.generate(PACKAGE, "imu_odometry_node", "ImuOdometry"))
